pkg/suggestion/v1beta1/chocolate/service.py

```diff
@@ -4,12 +4,15 @@
 from pkg.apis.manager.v1beta1.python import api_pb2
 from pkg.apis.manager.v1beta1.python import api_pb2_grpc
 
-from pkg.suggestion.v1beta1.internal.constant import DOUBLE
+from pkg.suggestion.v1beta1.internal.constant import (INTEGER, DOUBLE, CATEGORICAL, DISCRETE)
```

Comment:
I am not good at py, why do we need () for it?

Reply:
I think we can remove it from here, it uses when we need to use more than 1 line for importing